### ISSUE TITLE
Backfired spell's effect doesn't stop

### ISSUE THREAD

**Issue by Snowcreeper** [Electroblob77/Wizardry#449]
Minecraft version: 1.12.2
Wizardry version: 4.2.10
Spellpack for Twilight Forest: 1.0
Modpack: MC Eternal: 1.3.5.3
Environment: Singleplayer

Tried to uncover a spell (Spectral Pathway) without using s Scroll of Identification eventually backfired.
It gave me Weakness and Levitation, but the thing is the effect won't wear off.
Tried a bucket of milk, didn't help. Tried uncovering it with a proper Scroll of Indentification, didn't help. Am I doing something wrong or is it an actual bug? Do I have to die for it to go away?

Other mods involved: Don't know which mods could be relevant, I've stated the modpack above.

Link to crash report (if applicable): No crash, yet.

**Comment by Snowcreeper** [comment]
Update did have to die for it to stop :(

**Comment by Electroblob77** [comment]
It probably got desynced somehow, normally if this happens the effect doesn't _actually_ last forever, it's just the inventory icon that stays there. Did you try re-logging?

**Comment by Snowcreeper** [comment]
Yes I did try re-logging, and restarting too. Guess it somehow messed up, but it's a too low of a chance to happen, so maybe just leave it as is.

**Comment by Electroblob77** [comment]
Hmm, I'd still like to fix this if I can. I'll see if I can reproduce it. At least it's low-impact, from what you're saying.
